marv/_aggregate.py

```diff
@@ -189,8 +189,12 @@ def __getitem__(self, key):
         if key != 'meta' and not self.meta[key].has_output:
             return None
         node_json = os.path.join(self.path, '{}.json'.format(key))
-        with open(node_json) as f:
-            return json_load_agg(f)
+        try:
+            with open(node_json) as f:
+                return json_load_agg(f)
+        except IOError:
+            log.warn('File not found: %s', node_json)
```

Comment:
Printing the error itself is useful, `IOError` is also raised on permission denied for example.